stwcs/tests/test_astrometrydb.py

```diff
@@ -146,7 +143,28 @@ def test_success_offsets(self):
 
 
 def test_db_connection():
-
     adb = astrometry_utils.AstrometryDB()
     assert adb.available
     del adb
+
+
+def test_db_raise_errors_user_override():
+    # should not raise an exception since raise_errors is False
+    db = astrometry_utils.AstrometryDB(testing=True, url="bad_link/", raise_errors=False)
+    assert db.available is False
+    del db
```

Comment:
Can you add a test like this but passing `raise_errors=True` to test that overriding the env variable works.

Reply:
New test added in [65f4c8e1c6ffa22109fdb4073baaddb18269c94b](https://github.com/spacetelescope/stwcs/pull/231/commits/65f4c8e1c6ffa22109fdb4073baaddb18269c94b)